web/login.php

```diff
@@ -40,6 +40,27 @@
         exit;
     }
 
+    // Check for external authentication by web server
+    if (isset($_SERVER[$App->Config->configArr['extauth_remote_user']])) {
+	$Teams = new Teams($App->Users);
+
+	$fname = $_SERVER[$App->Config->configArr['extauth_first_name']];
```

Comment:
use `$App->Request->Server->get()` instead of global `$_SERVER`. See: https://symfony.com/doc/current/components/http_foundation.html#accessing-request-data

use firstname and longname instead of fname and lname to be consistant with the rest of the codebase.